content/docs/intro/console/collaboration/teams.md

```diff
@@ -16,12 +16,17 @@ The Pulumi Console offers role-based access control (RBAC) using teams.
 Teams allow organization admins to assign a set of stack permissions
 to a group of users.
 
-### Creating a Team
+## Team Roles
+
+Members of a team can be granted `Team admin` or `Team member` permissions. Team admin's can add members to a
+team. Both team admins and team members can grant stack access to a team. By default, any new team members will be
+assigned the team member role. To change a team member's role use the ellipsis menu item at the end of the table row.
 
-You can add a new team by going to the organization's **Teams** tab, and then
-clicking **New**. Only organization admins can create or update teams.
+### Creating a Team
 
-![Adding a new team](/images/docs/reference/service/new-team-card.png)
+Organization admins can add a new team by going to the organization's **Teams** tab and
+selecting **Create team**. Organization members can also be granted permissions to create teams
+in the organization settings. Any member who creates a team will automatically be assigned the team admin role.
```

Comment:
Should this maybe be

```suggestion
in the organization's Settings tab. Any member who creates a team will automatically be assigned the team admin role.
```

? It just reads a bit oddly as "in the organization settings."